examples/client.rs

```diff
@@ -266,7 +281,19 @@ fn main() {
                 },
             };
 
-            socket.send(&out[..write]).unwrap();
+            match socket.send(&out[..write]) {
```

Comment:
Ditto, this (and the similar matches in the other examples) can be simplfiied as:
```rust
if let Err(e) = socket.send(&out[..write]) {
    if e.kind() == std::io::ErrorKind::WouldBlock {
        debug!("send() would block");
        break;
    }

    panic!("send() failed: {:?}", e);
}
```

Reply:
Done